eazy/templates.py

```diff
@@ -1164,9 +1198,9 @@ def igm_absorption(self, z, scale_tau=1., pow=1):
         except:
             from eazy import igm as igm_module
         
-        igm = igm_module.Inoue14(scale_tau=scale_tau)
+        igm = igm_module.Asada24(sigmoid_params=sigmoid_params, scale_tau=scale_tau, add_cgm=add_cgm)
         igmz = self.wave*0.+1
-        lyman = self.wave < 1300
+        lyman = self.wave < 2000
```

Comment:
Could you add a switch like `max_fuv_wave = 2000 if add_cgm else 1300`?  It's a fairly minor thing, but perhaps gives a modest speed improvement.

Reply:
Thank you for your suggestion. I implemented the switch in commit 1a04d3a and 4dad304.